greenpeace/planet4-master-theme
# PLANET-6373 Blocks as a theme feature POC

Ref. https://jira.greenpeace.org/browse/PLANET-6373

- The Guestbook block is added in Master theme as POC of Blocks as a theme feature

**File/folder structure-** 
Backend-            
![image](https://github.com/greenpeace/planet4-master-theme/assets/5357471/add04400-65bd-4974-8828-a470334d2cd9)

Frontend-
![image](https://github.com/greenpeace/planet4-master-theme/assets/5357471/546d3fd6-d552-4086-b3ac-90ddf0ab7122)

Template-
![image](https://github.com/greenpeace/planet4-master-theme/assets/5357471/59044382-7b39-4ea0-bcc6-50feb3328a77)

**Testing**
- Deactivate the Blocks plugin
- Go to WP admin and add new page
- Add Guestbook block on page
- Publish the page and check Guestbook block on frontend
eg. https://www-dev.greenpeace.org/test-sinope/test-guestbook-block-as-a-part-of-mastertheme/
